src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  const getPartnerLabel = () => {
```

Comment:
You don't really need a function for it. Jus create a constant - `partnerLabel` for example.